webpack/assets/javascripts/react_app/components/Layout/layout.scss

```diff
@@ -2,6 +2,8 @@
 @import '../../common/variables.scss';
 
 .react-page #foreman-main-container {
+  overflow-x: auto;
+  white-space: nowrap;
```

Comment:
When its set to `nowrap` audits page can look weird for long entries (to test create a long bookmark in audits and refresh the page, and open the new bookmark entry).
Will this fix work with `white-space: normal`? I checked it in a few places and it looks fine

Reply:
You're right, I missed that. I changed it to `white-space: normal`, it looks fine to me, too. Thank you!